src/main/java/org/fineract/messagegateway/sms/data/EmailRequestDTO.java

```diff
@@ -0,0 +1,42 @@
+package org.fineract.messagegateway.sms.data;
+
+import java.util.List;
+import javax.validation.Valid;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+public class EmailRequestDTO {
+        @NotEmpty
+        private List< String> to;
+
+        @NotEmpty
+        private String subject;
+
+        @NotEmpty
```

Comment:
These annotation validations to be handled and sent in phee/fineract specific format.